bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/ScriptEngineManagerImpl.java

```diff
@@ -163,6 +166,17 @@ public void loadScript(String engineIdentifier, InputStreamReader scriptData) {
         }
     }
 
+    private void tryExtractAndInjectFilename(String engineIdentifier, ScriptEngine engine) {
+        try {
+            File maybeScriptFile = new File(engineIdentifier);
```

Comment:
> maybeScriptFile

How about just scriptFile?

Reply:
I can use that if you like; I called it this because it was a guess that it's the script file rather than something else?

Reply:
I think the method should be something like 'injectFilename' and the variable should be 'scriptFile'. I prefer to use definitive names... just my preference. Maybe the maintainers have an opinion?

Reply:
Ok, no worries; I've pushed those suggestions